src/components/TimeCardPage/TimeSheet.js

```diff
@@ -49,33 +47,91 @@ export default function Page() {
         // Callback for date-range picker - does any pre-processing when grabbing a new date 
         setStartDate(startDate); 
         setEndDate(endDate); 
+        currentTimesheetsToDisplay(userTimesheets, start); 
         //TODO - remove once we are finished setting up API calls for  this 
         console.log("New date range has been selceted:\n\t %s \nto \n\t%s", start, end); 
     }
     
-    const [userTimesheets,setTimesheets] = useState([]); 
+    const [userTimesheets, setTimesheets] = useState([]); 
```

Comment:
⛏️ By convention, useState names go `[x, setX]` so this should be `[userTimesheets, setTimesheets]`